core/server/worker/src/main/java/alluxio/worker/block/BlockMasterClient.java

```diff
@@ -69,7 +75,8 @@ protected long getServiceVersion() {
 
   @Override
   protected void afterConnect() throws IOException {
-    mClient = new BlockMasterWorkerService.Client(mProtocol);
+    // mClient = new BlockMasterWorkerService.Client(mProtocol);
```

Comment:
remove

Reply:
Being addressed as part of boot-up and security work.